README.md

```diff
@@ -174,6 +174,28 @@ for our complete browser support matrix:
 The polyfills may work in older browsers, however require additional polyfills (such as classList, or other [platform](https://github.com/webcomponents/webcomponents-platform)
 polyfills) to be used. We cannot guarantee support for browsers outside of our compatibility matrix.
 
+## Known Issues
+
+  * [Style encapsulation (inline styling in components) does not work out of the box](#shadycss)
+  * [Custom element's constructor property is unreliable](#constructor)
+  * [ShadyCSS: :host(.zot:not(.bar:nth-child(2))) doesn't work](#nestedparens)
+
+### Style encapsulation (inline styling in components) does not work out of the box. <a id="shadycss"></a>
+The ShadowDOM polyfill does not properly support CSS in ShadowDoM out of the box:
```

Comment:
inconsistent case: `ShadowDoM`, maybe do not need to mention it twice? how about:

> The ShadowDOM polyfill does not properly support scoped CSS in shadow trees out of the box: